infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -34,8 +30,10 @@
      * <p>
      * a cache of recent conflict keys and their latest global-log position.
      */
-    private final Cache<ConflictTxStream, Long> conflictCache;
-
+    private final Hashtable<ConflictTxStream, Long> cacheHashtable;
+    private final PriorityQueue<ConflictTxStreamEntry> cacheEntries; //sorted according to address
+    private final int cacheSize;
```

Comment:
why not long to make it consistent with the previous version of cache?

Reply:
The priorityQueue constructor only accept int as the argument.